torch/_torch_docs.py

```diff
@@ -2580,12 +2580,20 @@ def merge_dicts(*dicts):
 Returns the cross product of vectors in dimension :attr:`dim` of :attr:`input`
 and :attr:`other`.
 
-:attr:`input` and :attr:`other` must have the same size, and the size of their
-:attr:`dim` dimension should be 3.
+Supports input of float, double, cfloat and cdouble dtypes. Also supports batches
+of vectors, for which it computes the product along the dimension :attr:`dim`.
+In this case, the output has the same batch dimensions as the inputs.
 
 If :attr:`dim` is not given, it defaults to the first dimension found with the
 size 3. Note that this might be unexpected.
 
+.. seealso::
+        :func:`torch.linalg.cross` which has a -1 as the default dimension.
```

Comment:
"which requires specifying dim (defaulting to -1)"?

I think the principal difference to highlight is that linalg.cross doesn't have torch.cross's odd "search" behavior.